lib/constants/tool-names.ts

```diff
@@ -1,4 +1,5 @@
 export enum ToolName {
+    Testplane = 'testplane',
     Hermione = 'hermione',
```

Comment:
Will `Hermione` tool name still be used after this PR? As far as I can see, you've removed `hermione` from everywhere, maybe we can remove it from this place as well?